src/main/java/bio/terra/dao/MetaDao.java

```diff
@@ -1,11 +1,28 @@
 package bio.terra.dao;
 
+import org.springframework.jdbc.support.KeyHolder;
+
 import java.util.List;
+import java.util.Map;
+import java.util.UUID;
 
-public interface MetaDao<T> {
-    void create(T object);
-    T retrieve(String id);
+public abstract class MetaDao<T> {
+//    abstract UUID create(T object);
+//    T retrieve(String id);
     //void update(T object);
-    void delete(String id);
-    List<T> enumerate();
+//    void delete(String id);
+//    List<T> enumerate();
+
+    //TODO: find a better home for this, temporary fix for findBugs
```

Comment:
nit: This seems like a good home. Is the TODO still valid?

Reply:
I think it is a good home now, let's remove the TODO.

Reply:
i'm leaving the TODO here - we may still want to revisit.